perf/tests/renderComponent-basic.js

```diff
@@ -1,24 +1,22 @@
-/* jshint undef: true, unused: true */
-
-/* global document */
-/* global window */
-/* global Benchmark */
-/* global React */
-
-if (typeof exports == 'undefined') exports = {};
+if (typeof exports === 'undefined') {
+  exports = {};
+}
 
 /*http://benchmarkjs.com/docs#options*/
 
 exports.name = 'React.render single div';
 
 exports.setup = function(){
-  /*global*/_rootNode = document.createElement('div');
+  _rootNode = document.createElement('div');
   document.body.appendChild(_rootNode);
 };
 exports.fn = function(){
   React.render(React.DOM.div(null, 'lol, perf testing ', this.count), _rootNode);
 };
 exports.teardown = function(){
```

Comment:
Please do. For the most part, the rules we have in `src/.eslintrc` are the ones we want everywhere (though I see we don't have that one - perhaps we should. We can do that later since this is already big).